spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -596,15 +596,33 @@ function sendPacket(
     // increment the sequence. Thus there are monotonically increasing sequences for packet flow for a given clientId
     nextSequenceSend[sourceChannelId]=sequence+1
     
-    // log that a packet can be safely sent
-    // Event Emission 
-    emitLogEntry("sendPacket", {
+    // Event Emission for send packet
+    emitEvents("send_packet", {
       sourceId: sourceChannelId, 
-      destId: channel.counterpartyChannelId, 
-      sequence: sequence,
-      packet: packet,
-      timeoutTimestamp: timeoutTimestamp, 
+      destId: channel.counterpartyChannelId,
+      sequence: sequence, // value is string in decimal format
+      timeoutTimestamp: timeoutTimestamp, // value is string in decimal format
+      payloadLength: len(payloads), // value is string in decimal format
+      // include first payload data in events if there is only one payload
```

Comment:
Does that mean that if this is a multi-payload packet, these need to be empty strings?

Reply:
Right, yes because you would check the length to find out if there should be more to look for.